zetta_utils/convnet/simple_inference_runner.py

```diff
@@ -15,15 +17,16 @@ class SimpleInferenceRunner:  # pragma: no cover
     model_path: str
     unsqueeze_to: int | None = None
 
-    def __call__(self, src: torch.Tensor) -> torch.Tensor:
+    def __call__(self, src: npt.NDArray) -> npt.NDArray:
```

Comment:
At least the input should be our generic `Tensor`, in case the input already is a `torch.Tensor`?